src/garage/torch/policies/tanh_gaussian_mlp_policy.py

```diff
@@ -0,0 +1,184 @@
+"""TanhGaussianMLPPolicy."""
+import numpy as np
+import torch
+from torch import nn
+
+from garage.torch.modules import TanhGaussianMLPTwoHeadedModule
+from garage.torch.policies import Policy
+import garage.torch.utils as tu
+
+
+class TanhGaussianMLPPolicy(Policy, TanhGaussianMLPTwoHeadedModule):
+    """TanhGaussianMLPPolicy.
```

Comment:
please don't abuse the one-liner rule like this. your summary line should be an actual summary.